src/compiler/types.ts

```diff
@@ -3856,12 +3856,17 @@ namespace ts {
      */
     export type ResolvedConfigFileName = string & { _isResolvedConfigFileName: never };
 
+    export interface WriteFileCallbackData {
+        /*@internal*/ sourceMapUrlPos?: number;
+        /*@internal*/ buildInfo?: BuildInfo;

```

Comment:
This is to not have to re-parse json for buildInfo (used in tsbuild) to cache as well as get the dts change time